src/CMakeLists.txt

```diff
@@ -632,13 +634,24 @@ rocm_export_targets(
     ${PACKAGE_STATIC_DEPENDS}
 )
 
-
-
 # Install db files
-file(GLOB FIND_DB_FILES kernels/*.fdb.txt)
-install(FILES
-    kernels/miopen.db
-    ${FIND_DB_FILES}
- DESTINATION ${DATA_INSTALL_DIR}/db)
-
+if(NOT MIOPEN_EMBED_DB STREQUAL "")
+    include(embed)
+    set(CODE_OBJECTS)
+    list(APPEND CODE_OBJECTS "kernels/miopen.db")
```

Comment:
Why is this line here?

Reply:
Because the `MIOPEN_EMBED_DB` flag embeds both perf db and find db. This is the perf db part.